min_compatible_versions.csv

```diff
@@ -0,0 +1,189 @@
+This file was generated by scripts/min_compatible_versions.py
+pkg_name,min_version
+Flask-Cache,~=0.13.1
+Jinja2,~=2.11.0
+SQLAlchemy,==2.0.22
+WebTest,0
+Werkzeug,<1.0
+ai21,0
+aiobotocore,~=1.4.2
+aiofiles,0
+aiohttp,~=3.7
+aiohttp_jinja2,~=1.5.0
+aiomysql,~=0.1.0
+aiopg,~=0.16.0
+aiosqlite,0
+algoliasearch,~=2.5
+anthropic,~=0.28
+anyio,>=3.4.0
+aredis,0
+asgiref,~=3.0
+astunparse,0
+async_generator,~=1.10
+asyncpg,~=0.22.0
+asynctest,==0.13.0
+attrs,>=20
+austin-python,~=1.0

```

Comment:
this is not actually a dependency we should block on.

we use a version in our test suite for the `profile-diff` job, but it shouldn't matter what version of `austin-python` any of our customers are using.

also the test doesn't really tell us if we are compatible with `austin-python`, that is just the version we use in that test suite.

(same job, but same for `rich`)